tests.php

```diff
@@ -0,0 +1,114 @@
+<?php
+	/* Tests script for phpcs-security-audit */
+
+	//eval($_GET['a']);
+	echo "aaaa";
+	echo "bbbb" . $_POST['b'];
+	echo "b";
+	db_query($_GET['a']);
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> error - Potential SQL injection found in db_query()